tests/rados/test_add_db_wal_bst.py

```diff
@@ -0,0 +1,138 @@
+                        if not osd_config_result:
+                            log.error(
+                                f"The nvme path {dev_path} is not added to the osd"
+                            )
+                            return 1
+                        nvme_flag = True
+                    elif not ssd_flag:
+                        stop_osd(rados_object, test_osd)
+                        bluestore_obj.add_db_device(test_osd, dev_path)
+                        start_osd(rados_object, test_osd)
+                        osd_config_result = check_osd_config(
+                            rados_object, host_name, dev_path
+                        )
+                        if not osd_config_result:
+                            log.error(
+                                f"The SSD path {dev_path} is not added to the osd"
+                            )
+                            return 1
+                        ssd_flag = True
+
+                if nvme_flag:
+                    log.info(f"The NVMe is configured to the {test_osd}")
+                else:
+                    log.info(f"The NVMe is not configured to the {test_osd}")
+                if ssd_flag:
+                    log.info(f"The SSD is configured to the {test_osd}")
+                else:
+                    log.info(f"The SSD is not configured to the {test_osd}")
+                osd_flag_status = osd_flag_status + 1
+
+        if osd_flag_status == 0:
+            log.error("Any of the devices are not configured as OSDs")
+            return 1
+    except Exception as e:
+        log.info(e)
+        log.info(traceback.format_exc())
+        return 1
+    return 0
+
+
+def stop_osd(node_object, osd_id):
+    """
+    Method  used to stop the OSD
+    Args:
+        node_object: node object for calling the method
+        osd_id: osd id
+    Returns : none
+    """
+    if not node_object.change_osd_state(action="stop", target=osd_id):
+        log.error(f"Unable to stop the OSD : {osd_id}")
+        raise Exception("Execution error")
+
+
+def start_osd(node_object, osd_id):
+    """
+    Method  used to start the OSD
+    Args:
+        node_object: node object for calling the method
+        osd_id: osd id
+    Returns : none
+    """
+    if not node_object.change_osd_state(action="start", target=osd_id):
+        log.error(f"Unable to stop the OSD : {osd_id}")
+        raise Exception("Execution error")
+
+
+def check_osd_config(node_object, host_name, dev_path):
+    """
+    Method  used to start the OSD
+    Args:
+          node_object: node object for calling the method
+          host_name: host name in the cluster where the device need to check
+          dev_path: device path
+          Example: /dev/sdb
+    Returns : True/False
+    """
+    end_time = datetime.datetime.now() + datetime.timedelta(seconds=300)
+    while end_time > datetime.datetime.now():
+        volume_status = node_object.is_ssd_availabel(node_object, host_name, dev_path)
```

Comment:
Minor comment: The earlier merged function definition for "is_ssd_availabel" is used here. Can we fix the wording to - "is_ssd_available" in all files?

Reply:
I am using the same function which is available at "test_add_osd_ssd.py", So I created the common function "is_path_available" in the core_workflows.py".  The same is applicable for the "get_available_ssd" and "get_available_hdd" functions. I will remove the code from test_add_osd_ssd.py once this is merged.